<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePoint</class>
 <widget class="QWidget" name="CreatePoint">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>245</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>245</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/fish.png</normaloff>:/image/fish.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBoxTackle">
         <item>
          <property name="text">
           <string>Feeder Tackle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spinning Tackle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Float Tackle</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRigs">
         <property name="text">
          <string>Rigs:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textEditRigs"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxFrom">
         <item>
          <property name="text">
           <string>Shore Fishing</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Boat Fishing</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelFish">
         <property name="text">
          <string>Fish:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textEditFish"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelBaits">
         <property name="text">
          <string>Baits:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textEditBaits"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelNote">
         <property name="text">
          <string>Note:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textEditNote"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="labelFlyCasting">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>Fly Casting:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textEditFlyCasting">
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDistance">
         <property name="text">
          <string>Distance:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textEditDistance">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDepth">
         <property name="text">
          <string>Depth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textEditDepth">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelUnwantedFish">
         <property name="text">
          <string>Unwanted Fish:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="textEditUnwantedFish"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButtonDeletePoint">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete Point</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>198</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonConfirm">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CreatePoint</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
